suite-native/module-device-settings/src/components/DeviceFirmwareCard.tsx

```diff
@@ -80,49 +58,32 @@ export const DeviceFirmwareCard = () => {
 
             if (isUpgradable) {
                 return {
-                    title: (
-                        <Translation
-                            id="firmware.updateCard.newVersionAvailable"
-                            values={{ version: getFwUpdateVersion(device) }}
-                        />
-                    ),
+                    title: <Translation id="firmware.updateCard.newVersionAvailable" />,
                     variant: 'info',
                     buttonLabel: <Translation id="firmware.updateCard.updateButton" />,
-                    onButtonPress: () => {
-                        navigation.navigate(DeviceSettingsStackRoutes.ConfirmFirmwareUpdate);
-                    },
+                    onButtonPress: handleOnPress,
                     buttonProps: {
                         isDisabled: isDiscoveryRunning,
                         isLoading: isDiscoveryRunning,
                     },
                 } as const;
             }
-
-            return {
-                title: <Translation id="firmware.updateCard.upToDate" />,
-                variant: 'success',
-            } as const;
         }
 
         return undefined;
     })();
 
     return (
-        <CardWithIconLayout
+        <SettingsItemCard
             icon={deviceModelToIconName(deviceModel)}
             title={<Translation id="firmware.title" />}
             alertBoxProps={firmwareUpdateProps}
-        >
-            <HStack marginTop="sp12" spacing="sp2">
-                <FirmwareInfo
-                    label={<Translation id="firmware.version" />}
-                    value={firmwareVersion}
-                />
-                <FirmwareInfo
-                    label={<Translation id="firmware.type" />}
-                    value={<Translation id={firmwareTypeTranslationId} />}
-                />
-            </HStack>
-        </CardWithIconLayout>
+            subtitle={
+                <>
+                    <Translation id="firmware.version" /> {firmwareVersion}
+                </>
+            }
```

Comment:
nit: I suggest using a translation dynamic value here e.g.:
```suggestion
            subtitle={
                <Translation id="firmware.version" values={{ version: firmwareVersion }} />
            }
```

Reply:
Also I think that the translations in the `en.ts` file should be places at the same place as where it is used. So I believe at this place should be `id="moduleDeviceSettings.[whatever].firmware.version"` instead of `id="firmware.version"`

Reply:
[30081a8](https://github.com/trezor/trezor-suite/pull/19638/commits/30081a8d39fbbf327061d858c1f8811d2da67123)